plugins/kobolight.koplugin/main.lua

```diff
@@ -148,4 +159,39 @@ function KoboLight:onSwipe(_, ges)
     return true
 end
 
+function KoboLight:addToMainMenu(tab_item_table)
+    table.insert(tab_item_table.plugins, {
+        text = _("Frontlight gesture controller"),
+        callback = function()
+            local image = ImageWidget:new{
+                file = PluginLoader.plugin_path .. "/kobolight.koplugin/demo.png",
+                height = Screen:getHeight(),
+                width = Screen:getWidth(),
+                scale_factor = 0,
+            }
+            UIManager:show(image)
+            UIManager:show(ConfirmBox:new{
+                text = T(_("Frontlight gesture controller can:\n- Turn on or off frontlight by tapping left-bottom of the screen.\n- Change frontlight intensity by swiping up or down on the left of the screen.\n\nDo you want to %1 it?"),
+                         self:disabled() and _("enable") or _("disable")),
+                ok_text = self:disabled() and _("Enable") or _("Disable"),
```

Comment:
The second button is "Close". It gives me an impression to "Close this feature" if the other button is "Yes", which is not so clear. But I cannot find another better word to replace the "Close" button.
But if the question is "Do you want to enable it?" and the button is "Enable", it's pretty clear this button is to "Enable" or "Disable" this feature.